android/app/src/main/java/com/mattermost/MainApplication.java

```diff
@@ -4,6 +4,8 @@
 import android.util.Log;
 
 import com.facebook.react.ReactApplication;
+import com.psykar.cookiemanager.CookieManagerPackage;
```

Comment:
Which of these are we actually using? I'm amazed the java compiler doesn't freak out at this due to the same class name being imported twice